python/core/raster/qgscolorrampshader.sip

```diff
@@ -42,9 +50,6 @@ class QgsColorRampShader : QgsRasterShaderFunction
     /** \brief Get the color ramp type as a string */
     QString colorRampTypeAsQString();
 
```

Comment:
Don't remove methods in core classes. They can be deprecated and if possible routed to a corresponding replacement.
They will be removed with the next major release (QGIS 3)